contracts/Airdrop.sol

```diff
@@ -14,11 +14,16 @@ contract Airdrop is VestingPool {
     // Time until which the airdrop can be redeemed
     uint64 public immutable redeemDeadline;
 
+    /// @notice Creates the airdrop for the token at address `_token` and `_manager` as the manager. The airdrop can be redeemed until `_redeemDeadline`.
+    /// @param _token The token that should be used for the airdrop
+    /// @param _manager The manager of this airdrop (e.g. the address that can call `initializeRoot`)
```

Comment:
minor: You probably mean "i.e." instead of "e.g." (German "d.h." vs. "z.B.")

Reply:
actually in this case `e.g.` is more fitting, as it is the only thing they can do :D

Reply:
with `e.g.` I would think also another address can be set as manager. But either way, nitpicking ;)